site/data/refDataApi.ts

```diff
@@ -39,7 +39,31 @@ export const fetchStops = async (input: FetchStopsInput) => {
         return [];
     }
 
-    return parseResult.data;
+    const filteredStopsData = parseResult.data.filter((stop) => {
+        if (
+            stop.stopType === "BCT" &&
```

Comment:
Could we pass these values to the api call as query params which is later used to query only these records instead of filtering stops after obtaining all the stops in the db

Reply:
yes in another ticket for the reference service improvements

Reply:
done